sycl/test-e2e/AddressSanitizer/bad-free/bad-free-host.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: linux
 // RUN: %{build} %device_asan_flags -O0 -g -o %t
-// RUN: %force_device_asan_rt %{run} not %t 2>&1 | FileCheck %s
+// RUN: %force_device_asan_rt %{run} not %t 2>&1 FileCheck %s
```

Comment:
can you please explain this modification?

Reply:
Joining the conversation, as `SYCLcompat` tests also present the same changes.

Reply:
That was a wrong decision on making the PR ready for review, this was only for testing purposes.